src/main/java/leo/client/ClientLoginDialog.java

```diff
@@ -368,6 +381,12 @@ public void itemStateChanged(ItemEvent e) {
             createButton.setEnabled(remote);
         }
 
+        if (e.getSource() == savePasswordCheckbox) {
```

Comment:
I think this is the only place where it's potentially set to true - which causes one issue: when you restart the game with already saved password, checkbox is ticked, but `rememberPassword` is set to `false`, and the password won't get saved (unless you toggle the checkbox again).